app/assets/javascripts/member-facing.js.erb

```diff
@@ -19,6 +19,8 @@
 require("<%= Settings.external_js_path %>");
 <% end %>
 
+window.URI = require('urijs');
```

Comment:
Okay! Let's use it then, I'm open to suggestions on how we want to handle javascript dependencies from `sou-assets`. Although I'm feeling we should leave conversation for another time